packages/react/src/skeleton/skeleton.stories.tsx

```diff
@@ -130,26 +130,26 @@ export const Article: Story = {
         A subtitle with some more text is loading
       </Skeleton>
       <p>
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(25, 60)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(45, 80)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(25, 60)} />
+        <Skeleton variant="text" width={numberBetween(45, 80)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
       </p>
 
       <Skeleton variant="text" as="h2">
         Another piece of text
       </Skeleton>
       <p>
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(25, 60)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(45, 80)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
-        <Skeleton variant="text" width={`${numberBetween(75, 90)}%`} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(25, 60)} />
+        <Skeleton variant="text" width={numberBetween(45, 80)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
+        <Skeleton variant="text" width={numberBetween(75, 90)} />
```

Comment:
In this example I wanted the width to be actual percentage of the full width. Maybe the api needs a little bit of tinkering 🤔 

You see here the style is just passed down directly: https://github.com/bring/hedwig-design-system/blob/master/packages/react/src/skeleton/skeleton.tsx#L54-L62

Wouldn't this change make it so the widths become something between 75px and 90px?

Reply:
💭 Could make the function be called `percentageBetween(min, max)` instead 🤔

Reply:
Oh, my bad! Didn't notice the % sign! Will change